app/react/Notifications/components/Notification.js

```diff
@@ -21,7 +21,14 @@ export class Notification extends Component {
     return (
       <div className={cssClass} onClick={this.close.bind(this)}>
         <Icon icon={icon} />
-        <span>{message}</span>
+        <span className="alert-text">
+          {message.split('\n').map((item, i) => (
+            <React.Fragment key={i}>
```

Comment:
@txau supress this error or use uniqueID form shared/uniqueID.js

Reply:
Supressed